addons/sys_zeus/XEH_postInit.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+
+private _category = format ["ACRE2 %1", localize "str_a3_cfghints_curator_curator_displayname"];
+
+[_category, "ZeusTalkFromCamera",  [(LLSTRING(SpeakFromCamera)), (LLSTRING(SpeakFromCamera_description))],
+    { call FUNC(handleZeusSpeakPress) },
+    { call FUNC(handleZeusSpeakPressUp) },
+[41, [false, false, false]]] call CBA_fnc_addKeybind; //Default bound to `
+
+if (hasInterface && {isClass (configFile >> "CfgPatches" >> "ace_interact_menu")}) then {
+    private _acreNode = ["ACRE_ZeusEars", "ACRE", "", {}, { true }] call ace_interact_menu_fnc_createAction;
```

Comment:
"ACRE" could be "Ears" or something like that. Depends if we want to add more stuff I guess.